tests/test_ensemble.py

```diff
@@ -0,0 +1,203 @@
+# Copyright © 2021 United States Government as represented by the Administrator of the
+# National Aeronautics and Space Administration.  All Rights Reserved.
+
+from io import StringIO
+import numpy as np
+import sys
+import unittest
+
+from prog_models import EnsembleModel
+from prog_models.models.test_models.linear_models import OneInputOneOutputOneEventLM, OneInputOneOutputOneEventAltLM
+
+
+class TestEnsemble(unittest.TestCase):
+    def setUp(self):
+        # set stdout (so it wont print)
+        sys.stdout = StringIO()
+
+    def tearDown(self):
+        sys.stdout = sys.__stdout__
+
+    def test_single_model(self):
+        # An ensemble model with one model should raise an exception
+
+        m = OneInputOneOutputOneEventLM()
+        
+        with self.assertRaises(ValueError):
+            EnsembleModel([m])
+
+    def test_wrong_type(self):
+        # An ensemble model with a non-model should raise an exception
+
+        m = OneInputOneOutputOneEventLM()
+        with self.assertRaises(TypeError):
+            EnsembleModel(m)
+        with self.assertRaises(TypeError):
+            EnsembleModel([m, 1])
+        with self.assertRaises(TypeError):
+            EnsembleModel(77)
+        with self.assertRaises(TypeError):
+            EnsembleModel([m, m, m, 77])
+
+    def test_two_models_identical(self):
+        """
+        this tests that the ensemble model works with two identical model-types with identical states, inputs, etc., one with slightly altered parameters
+        """
+        m = OneInputOneOutputOneEventLM()
+        m2 = OneInputOneOutputOneEventLM(x0={'x1': 2})
+
+        # Make sure they're not the same - 3x the impact
```

Comment:
maybe stating "they're not the same - 3x their original values" would be a bit clearer